ffs/api/api.go

```diff
@@ -96,9 +106,43 @@ func (i *API) ID() ffs.APIID {
 	return i.cfg.ID
 }
 
-// WalletAddr returns the Lotus wallet address.
-func (i *API) WalletAddr() string {
-	return i.cfg.WalletAddr
+// WalletAddr returns the wallet addresses.
+func (i *API) WalletAddrs() map[string]AddrInfo {
+	return i.cfg.Addrs
```

Comment:
I think it would be better to return a copy of the map (guarded by locks).
Just in case the caller won't be messing with the same internal map and do some nasty thing.
Compared to other things this happens because `map`s are reference types.

Reply:
Cool, done.